launchUI.py

```diff
@@ -2143,10 +2143,15 @@ def create_layout_pannel(master, x, width, height):
 
 def clickDisplayLayout():
     # Hardcodes the related path, will fix later.
-    flow_basePath = os.path.dirname(os.path.abspath(__file__)) + "/tools/OpenROAD-flow-scripts/flow/"
-    layout_path = flow_basePath + "layout.png"
-    odb_path = flow_basePath + "results/nangate45/gcd/base/6_final.odb"
-    cmd_path = flow_basePath + "cmd.tcl"
+    test_module_name = "TileRTL__f6add65c4dde1319"
+    test_platform_name = "asap7"

```

Comment:
Hmmm, asap7nm is open-source, then there is no reason people still use 45nm, right?

Reply:
Nangate45 is also open-source, I use asap7 here only because Deepak uses it in his Google Doc. Maybe there should a button on the GUI that allows user to choose what process to use?

Reply:
Cool, do you have bandwidth to add two selector (7nm, 45nm) for the users in that panel?

Reply:
Do you mean that I have time to do this? Yeah sure.

Reply:
Based on my recent review experience, reviewers are starting to judge on the nm technology, and view 45nm as outdated. People also say there is open-source 28nm lib, can you help to see whether OpenRoad has 28nm support? If no, the two selectors (one for 7nm, and one for 45nm) would be good enough. Thanks!

Reply:
Yeah sure.